protos/perfetto/trace/track_event/counter_descriptor.proto

```diff
@@ -92,4 +92,8 @@ message CounterDescriptor {
 
   // TODO(eseckler): Support arguments describing the counter (?).
   // repeated DebugAnnotation debug_annotations;
+
+  // All counter tracks with the same |y_axis_share_key| and the same parent
```

Comment:
Please improve the comments here. Be verbose and explain the terminology of what you are doing. Add motivation for why someone may want to use this